### PR TITLE
Update web-sockets.md

### PR DESCRIPTION
**Pull request by kimuson**
The source code is not compatible with the latest Flutter writing style, and we suggest changes.

_Description of what this PR is changing or adding, and why:_

_Issues fixed by this PR (if any):_

## Presubmit checklist
- [ ] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [ ] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [ ] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/master/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/cookbook/networking/web-sockets.md [hunk 1]
```diff
@@ -116,11 +116,9 @@ channel.sink.close();
 import 'package:flutter/material.dart';
 import 'package:web_socket_channel/web_socket_channel.dart';
 
-void main() => runApp(const MyApp());
+void main() => runApp(MyApp());
 
 class MyApp extends StatelessWidget {
-  const MyApp({super.key});
```
Comment (domesticmouse) [inline src/cookbook/networking/web-sockets.md:122]: Why are you deleting this?

File: src/cookbook/networking/web-sockets.md [hunk 2]
```diff
@@ -135,9 +133,9 @@ class MyApp extends StatelessWidget {
 
 class MyHomePage extends StatefulWidget {
   const MyHomePage({
-    super.key,
```
Comment (domesticmouse) [inline src/cookbook/networking/web-sockets.md:138]: This looks like a regression, using super parameters is something we want to encourage.

**Review by domesticmouse — CHANGES_REQUESTED** [review]
You are modifying code in the `web-socets.md` without changing the underlying code excerpt source file. If you had, you'd see that you get linter warnings for two of your changes.

I'm curious what problem this PR is attempting to solve.

**Comment by sfshaza2** [comment]
@kimuson, thanks for your PR, but it introduces a regression and doesn't match our source code guidelines. I'm closing this PR.

**Pull request closed without merging**